packages/console/src/i18n/init.ts

```diff
@@ -14,11 +13,11 @@ const initI18n = async (language?: Language) =>
       interpolation: {
         escapeValue: false,
       },
+      lng: language,
```

Comment:
didn't get this one. looks like the API preference is out of sync with console settings. why this can fix?